.github/workflows/CICD.yml

```diff
@@ -206,7 +206,7 @@ jobs:
         # target-specific options
         # * CARGO_FEATURES_OPTION
         CARGO_FEATURES_OPTION='--all-features' ;
-        if [ -n "${{ matrix.job.features }}" ]; then CARGO_FEATURES_OPTION='--features "${{ matrix.job.features }}"' ; fi
+        if [ -n "${{ matrix.job.features }}" ]; then CARGO_FEATURES_OPTION='--features ${{ matrix.job.features }}' ; fi
```

Comment:
Why remove the quotes?

Reply:
If I didn't remove the quotes, the CI is failing to get utilities, as you can see here. https://github.com/miles170/coreutils/actions/runs/4516836207/jobs/7955560726